k8s/options

```diff
@@ -0,0 +1 @@
+--argumentsRealm.passwd.jenkins=CHANGE_ME --argumentsRealm.roles.jenkins=admin
```

Comment:
Is the intention that this gets changed locally? My strategy has generally been to use .md files to document what local configuration needs to be created, and not check it in. If we do want to check it in, maybe checking in `options.example` but actually consuming `options` would be better.

Reply:
(this file is in the context of the Jenkins GKE guide, see above)